utilities/test_suite/HIP/runTests.py

```diff
@@ -271,17 +275,17 @@ def rpp_test_suite_parser_and_validator():
 os.chdir(buildFolderPath + "/build")
 
 # Run cmake and make commands
-subprocess.run(["cmake", scriptPath], cwd=".")   # nosec
-subprocess.run(["make", "-j16"], cwd=".")    # nosec
+subprocess.call(["cmake", scriptPath], cwd=".")   # nosec
+subprocess.call(["make", "-j16"], cwd=".")    # nosec
 
 # List of cases supported
 supportedCaseList = ['0', '1', '2', '4', '8', '13', '20', '21', '23', '29', '30', '31', '32', '33', '34', '35', '36', '37', '38', '39', '45', '46', '54', '61', '63', '65', '68', '70', '79', '80', '82', '83', '84', '85', '86', '87', '88', '89', '90', '91', '92']
 
 # Create folders based on testType and profilingOption
 if testType == 1 and profilingOption == "YES":
-    os.makedirs(f"{dstPath}/Tensor_PKD3")
-    os.makedirs(f"{dstPath}/Tensor_PLN1")
-    os.makedirs(f"{dstPath}/Tensor_PLN3")
+    os.makedirs("{}/Tensor_PKD3".format(dstPath))
+    os.makedirs("{}/Tensor_PLN1".format(dstPath))
```

Comment:
Doesn't string concatenation work for all Python versions? Like using (dstPath + "/Tensor_PKD3")? 
Or is the .format() useful somewhere else?

Reply:
changed to concatenation